testcases/PowerNVDump.py

```diff
@@ -305,6 +309,76 @@ def runTest(self):
         return boot_type
 
 
+class SBECrash_MPIPL(PowerNVDump):
+
+    def runTest(self):
+
+        if "OpenBMC" in self.bmc_type:
+            cmd = "mboxctl --reset"
+        elif "SMC" in self.bmc_type:
+            raise self.skipTest(
+                "SBE initiated MPIPL is not supported on SMC yet")
+        #run mboxctl reset on BMC
```

Comment:
Needs better comment here. 
BMC is not aware of MPIPL. We have to reset MBOX so that it can point to initial flash area and SBE can load it from LPC bus.

Reply:
Sure.